test/repl.jl

```diff
@@ -100,6 +100,13 @@ if !is_windows() || Sys.windows_version() >= Sys.WINDOWS_VISTA_VER
     end
     cd(origpwd)
 
+    # issue #20482
+    write(stdin_write, ";")
+    readuntil(stdout_read, "shell> ")
+    write(stdin_write, "echo hello >/dev/null\n")
```

Comment:
since windows shell-mode executes commands directly rather than going through a shell, I get
```
        From worker 3:  hello >/dev/null
show (2)                  |   19.83  |  0.32  |  1.6 | 369.99     | 1455.00
repl (3)                  |    9.42  |  0.16  |  1.7 | 178.81     | 1105.19
```
from this so I think this should be unix-only